runclient-Tools.sh

```diff
@@ -0,0 +1,2 @@
+dotnet run --project Content.Client --configuration Tools
```

Comment:
you need shebang to run it as `./runclient-Tools.sh` anyway instead of `sh runclient-Tools.sh`

Reply:
Point taken that is good practise, though @deltanedas 
![image](https://github.com/user-attachments/assets/03888f43-120a-4be1-88ec-0e1e144d67e5)
Does this depend on distro because I'm able to run it with ./ without the shebang